src/components/scripture/reference.ts

```diff
@@ -124,3 +124,27 @@ export const bookCodeToName = (code: string) => {
   }
   return book.names[0];
 };
+// return random scriptureRefenence as an array
+export const createRandomScriptureReferences = (): ScriptureRangeInput[] => {
+  //To do
+  // const book = books[Math.floor(Math.random() * books.length)];
+  const book = books[Math.floor(Math.random() * 2)];
+  // const startChapter = book.chapters[Math.floor(Math.random() * book.chapters.length)];
+  const startChapter = book.chapters[Math.floor(Math.random() * 2)];
+  // const endChapter = book.chapters[Math.floor(Math.random() * book.chapters.length)];
+  const endChapter = book.chapters[Math.floor(Math.random() * 2)];
```

Comment:
lodash's `random` might be easier to work with. There's also `faker.random.arrayElement`